api/src/main/java/com/codeforcommunity/rest/FailureHandler.java

```diff
@@ -1,22 +1,6 @@
 package com.codeforcommunity.rest;
 
-import com.codeforcommunity.exceptions.CreateUserException;
-import com.codeforcommunity.exceptions.EmailAlreadyInUseException;
-import com.codeforcommunity.exceptions.ExpiredSecretKeyException;
-import com.codeforcommunity.exceptions.HandledException;
-import com.codeforcommunity.exceptions.InvalidSecretKeyException;
-import com.codeforcommunity.exceptions.MalformedParameterException;
-import com.codeforcommunity.exceptions.MissingHeaderException;
-import com.codeforcommunity.exceptions.MissingParameterException;
-import com.codeforcommunity.exceptions.SchoolAlreadyExistsException;
-import com.codeforcommunity.exceptions.SchoolContactAlreadyExistsException;
-import com.codeforcommunity.exceptions.SchoolContactDoesNotExistException;
-import com.codeforcommunity.exceptions.SchoolDoesNotExistException;
-import com.codeforcommunity.exceptions.TokenInvalidException;
-import com.codeforcommunity.exceptions.UnknownCountryException;
-import com.codeforcommunity.exceptions.UsedSecretKeyException;
-import com.codeforcommunity.exceptions.UserDoesNotExistException;
-import com.codeforcommunity.exceptions.UsernameAlreadyInUseException;
+import com.codeforcommunity.exceptions.*;
```

Comment:
If we are using everything inside the exceptions directory then this should be okay but if not I think naming each individual class would be better to not load any classes that we are not going to be using

Reply:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports
@DerekL27